index.bs

```diff
@@ -6815,7 +6825,7 @@ may return any value, which will be discarded.
         1.  Round |x| to the nearest integer, choosing the even integer if it lies halfway between two,
             and choosing +0 rather than −0.
         1.  Return |x|.
-    1.  If |x| is <emu-val>NaN</emu-val>, +0, −0, +∞, or −∞,
+    1.  If |x| is <emu-val>NaN</emu-val>, +∞, or −∞,
```

Comment:
You're right. This should be 0 to be consistant with turning both −0 and +0 to 0, and +0 otherwise.